### DISCUSSION TITLE
GP-Engineer + Open Interpreter

### DISCUSSION THREAD

**Discussion by VenTheZone** [AntonOsika/gpt-engineer#729] [Ideas]
Alright so hear me out, got a big brain.
The dev. Agents created by gpt engineers.

What if they become agents which each their own Code Interpreter?

**Comment by ATheorell** [comment]
Hi,
Sounds intriguing! 
Can you elaborate a little? 
Do you see this as a threat or an opportunity to do X?

**Reply by VenTheZone** [reply]
Just an idea to make those GPT engineer more.... calculative and precise.
By integrating Code/Open Interpreter onto GPT engineer
